ops/ccs-ops-misc/load_test/v2/partially_adjudicated.py

```diff
@@ -1,78 +1,55 @@
-import urllib3
-
 from locust import HttpUser, task, tag
+from common.pac_tests import PACTestUser
```

Comment:
@KyRobbins what's the advantage of splitting the logic up between `partially_adjudicated` and `pac_tests`?